regression-test/suites/load_p0/routine_load/test_routine_load.groovy

```diff
@@ -1168,4 +1168,53 @@ suite("test_routine_load","p0") {
             j++
         }
     }
+
+    // partial
+    if (enabled != null && enabled.equalsIgnoreCase("true")) {
+        try {
+
+            sql new File("""${context.file.parent}/ddl/uniq_tbl_partial_drop.sql""").text
+            sql new File("""${context.file.parent}/ddl/uniq_tbl_partial_create.sql""").text
+            sql new File("""${context.file.parent}/ddl/uniq_tbl_partial_init.sql""").text
+
+            sql """
+                CREATE ROUTINE LOAD job_uniq_tbl_partial ON routine_load_uniq_tbl_partial
+                COLUMNS(k00,k01,k02,k03,k04,k05,k06,k07,k08,k09,k10,k11,k12,k13,k14,k15,k16,k17,k18),
```

Comment:
Please check your column mapping, the number of columns in the source file is greater than the actual number of table columns https://doris.apache.org/docs/data-operate/import/import-scenes/load-data-convert/#:~:text=There%20are%20more%20columns%20in%20the%20source%20file%20than%20in%20the%20table

Reply:
> source

In fact, There are as many fields in the table as there are in the file, I just want to provide a subset of the columns so that partial column updates can be done. I configured it like this, but it will be abnormal. How do I configure the update of some columns?

Reply:
I can't see the latest exception, but the error with this Commit is due to a problem with the column mapping. https://github.com/apache/doris/pull/27234/commits/e1cbcdca43de2b35466032e571cd09c3ffe95e3a